theories/crypto/PKE.ec

```diff
@@ -163,3 +164,71 @@ module Correctness (S:Scheme) = {
     return (m' = Some m);
   }
 }.
+
+module type CAdversary = {
```

Comment:
Is this C (and the others below) for "classical"? We might want to avoid adding it in until merge of quantum.

This gives me ideas for better structuring the crypto libraries and avoid massive files.

Reply:
It is Correctness Adversary i believe.

Reply:
Oh, I see. This is for capturing cases where the set of valid plaintexts might depend on the key pair, so we can't simply quantify universally over all plaintexts. Looks good to me here.